docs/moduleCom7Regional.rst

```diff
@@ -0,0 +1,316 @@
+
+1. Individual avalanche directories for each group containing:
+
+  - Clipped DEM file
+  - Scenario-specific release area shapefiles
+  - Optional: Clipped entrainment and resistance areas
+
+2. Two visual reports (see :numref:`fig-splitInputs-basic` and :numref:`fig-splitInputs-optional`)
+
+3. Scenario report in txt format (see example below)
+
+.. list-table::
+   :widths: 50 50
+
+   * - .. _fig-splitInputs-basic:
+       .. figure:: /_static/splitInputs_visualReport_basic.png
+          :width: 100%
+          :alt: Basic visual report
+
+          Example of basic inputs report displaying resulting groups and their extent
+     - .. _fig-splitInputs-optional:
+       .. figure:: /_static/splitInputs_visualReport_optional.png
+          :width: 100%
+          :alt: Optional inputs report
+
+          Example of optional inputs report displaying RES and ENT areas for each group
+
+Example Scenario Report:
+
+.. code-block:: text
+
+    SCENARIO REPORT
+    ==============
+    Generated: 2025-02-04 10:58:56
+
+    Group: group1
+    ------------
+
+    Scenario: sce1
+    No. of release areas: 2
+    - rel1
+    - rel2
+
+    No. of entrainment areas: 1
+    No. of resistance areas: 3
+
+    Group: group2
+    ------------
+    ...
+
+Configuration
+-------------
+Settings are controlled through ``splitInputsCfg.ini``, in which the ``bufferSize`` for the group extent is defined (which is used for DEM, RES, and ENT clipping into
+smaller chunks). By default, this value is set to 2500 m. For each group, a bounding box is created from the maximum x-y extent of all release features in the group.
+The value for ``bufferSize`` is then added to each direction (+x, -x, +y, -y). This buffer may be adjusted according to the expected maximum runout length of your avalanches -
+a larger value will ensure that no simulation will exit its domain, but with the drawback of producing larger output files.
+
+Algorithm
+---------
+The ``splitInputsMain`` function, which is called in ``runScripts/runSplitInputs.py``, performs the following steps:
+
+1. Create central avalanche directory list
+2. Set up avalanche directories
+3. Split and write release areas to each directory
+4. Clip and write DEM to each directory
+5. Clip and write optional input to each directory (currently includes RES and ENT)
+6. Divide release areas into scenarios
+7. Write reports
+
+To Run - Script based
+---------------------
+1. Prepare inputs in your ``<avalancheDir>/Inputs``
+2. Configure settings in ``splitInputsCfg.ini`` (or local version ``local_splitInputsCfg.ini``)
+3. Either set path to avalanche directory in ``avaframeCfg.ini`` (or local version ``local_avaframeCfg.ini``) or
+   call command below with the avalanche directory as argument
+4. Execute from ``AvaFrame/avaframe`` directory:
+
+.. code-block:: bash
+
+    python runScripts/runSplitInputs.py
+
+---------------
+
+Running multiple avalanche dirs
```

Comment:
as it is a heading, shall we call it avalanche directories? or to be consistent if this is required here avalancheDirs?